.circleci/config.yml

```diff
@@ -29,7 +29,10 @@
               python scripts/download_glue_data.py --data_dir data --tasks all
               source activate jiant
               python -m nltk.downloader perluniprops  nonbreaking_prefixes punkt
-              python main.py --config_file config/demo.conf
+              python main.py --config_file config/demo.conf --overrides "transfer_paradigm=finetune"
```

Comment:
Ignore these changes - this is just a way to automatically test that all of these cases pass. This will not be pushed to master.

Reply:
Sounds good. I made a parallel proposal to add one of these cases permanently.